javaparser-core/src/main/javacc_support/com/github/javaparser/GeneratedJavaParserSupport.java

```diff
@@ -88,23 +86,31 @@ static void addMultipleModifier(GeneratedJavaParser generatedJavaParser, EnumSet
             modifiers.add(m);
     }
 
+    static TokenRange range(JavaToken begin, JavaToken end) {
```

Comment:
That would make the grammar more verbose - this method is called in every `new` :-O I think I should be able to clean up some stuff here, but that's for another day.